Module/client/src/components/app.jsx

```diff
@@ -1,15 +1,32 @@
 import React from 'react';
+import axios from 'axios';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      // refers to the id number typed in the url
+      id: window.location.href.match(/id\s*=\s*(.*)/)[1] - 1,
```

Comment:
What happens if your state.id is null? If there is no match? (i.e. id=459). Might want to handle invalid inputs.

Reply:
It now automatically sets the id to one if the input is invalid.